api/init/security/decorators.py

```diff
@@ -0,0 +1,40 @@
+from flask import request, make_response, jsonify
+from security.token import is_token_valid
+
+
+AUTHORIZATION_HEADER = 'Authorization'
+JWT_PREFIX = 'Bearer '
+
+
+def token_required(func):
+    """Gets a decorator to check whether a valid token is in the request headers. It will return
+        401, if the user provdies an invalid token.
+    """
+    def wrapper(*args, **kwargs):
+        auth_header = _get_token_from_header()
+        token = _parse_jwt_token(auth_header)
+        unauthorized_response = make_response(jsonify({}), 401)
```

Comment:
Add a message to the return body

Reply:
See 0aafaafba016b3ecb6d62307026422341f1e620a